modules/hybridBidAdapter.js

```diff
@@ -94,6 +97,35 @@ function buildBid(bidData) {
         bid.renderer = createRenderer(bid);
       }
     }
+  } else if (bidData.placement === PLACEMENT_TYPE_IN_IMAGE) {
+    bid.mediaType = BANNER;
+    bid.inImageContent = {
+      content: {
+        content: bidData.content,
+        actionUrls: {}
+      }
+    };
+    let actionUrls = bid.inImageContent.content.actionUrls;
+    actionUrls.loadUrls = bidData.inImage.loadtrackers || [];
+    actionUrls.impressionUrls = bidData.inImage.imptrackers || [];
+    actionUrls.scrollActUrls = bidData.inImage.startvisibilitytrackers || [];
+    actionUrls.viewUrls = bidData.inImage.viewtrackers || [];
+    actionUrls.stopAnimationUrls = bidData.inImage.stopanimationtrackers || [];
+    actionUrls.closeBannerUrls = bidData.inImage.closebannertrackers || [];
+
+    bidData.inImage.but = true;
+
+    if (bidData.inImage.but) {
```

Comment:
@hybrid-ai what is the purpose of this `if` when you set this param to true on L116?

Reply:
@aleksatr This is an additional button for inImage banner and that button has its own styles

Reply:
@hybrid-ai , ok, but you set `bidData.inImage.but` to true on L116, so `if` statement on L118 is not needed, it will always evaluate to `true`

Reply:
@aleksatr I am sorry. I added it for debug but I forget to delete this line. Thank you for your comment.